helm/examples/esdl-sandbox/README.md

```diff
@@ -0,0 +1,41 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+Example of esdl-sandbox service.  This service is an alpha release.  It may change quite a bit before the final release.
+
+After deploying a helm k8s hpcc..
+
+From helm/examples/esdl-sandbox folder:
```

Comment:
Does this require the user to ssh into the node running the esp ?

Reply:
Hmmm.. these examples are from the perspective of whoever is deploying has a local copy of the helm stuff.  I'll be working out a more concrete example for the conference and may put it somewhere.

Reply:
No, this is more intended for someone playing with helm deployments for now.  I'm going to come up with a better demo, record it, and maybe put the files somewhere else.

Reply:
That is, after this PR.